xbmc/utils/EGLUtils.cpp

```diff
@@ -112,7 +173,16 @@ bool CEGLUtils::HasClientExtension(const std::string& name)
 
 void CEGLUtils::LogError(const std::string& what)
 {
-  CLog::Log(LOGERROR, "%s (EGL error %d)", what.c_str(), eglGetError());
+  EGLenum error = eglGetError();
+
+  for (const auto& eglError : eglErrors)
+  {
+    if (eglError.first == error)
+    {
+      CLog::Log(LOGERROR, "{} ({})", what.c_str(), eglError.second);
```

Comment:
Would be better to log the code in hex in case it's not in the array

Reply:
I changed it so that the hex will be logged if the matching string isn't found